docs/sources/reference/run.md

```diff
@@ -426,36 +426,37 @@ the `--security-opt` flag. For example, you can specify the MCS/MLS level, a
 requirement for MLS systems. Specifying the level in the following command
 allows you to share the same content between containers.
 
-    # docker run --security-opt label:level:s0:c100,c200 -i -t fedora bash
+    $ sudo docker run --security-opt label:level:s0:c100,c200 -i -t fedora bash
```

Comment:
Why was this changed from being implicit with `#` to `sudo`? Is that consistent with the rest of the docs?